lib/matplotlib/backends/backend_qt5.py

```diff
@@ -9,8 +14,15 @@
     FigureCanvasBase,  FigureManagerBase, MouseButton, NavigationToolbar2,
     TimerBase, ToolContainerBase, figureoptions, Gcf
 )
+from . import backend_qt as _backend_qt  # noqa
 
 
 @_BackendQT.export
 class _BackendQT5(_BackendQT):
     pass
+
+
+def __getattr__(name):
+    if name == 'qApp':
+        return _backend_qt.qApp
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Comment:
Why expose a new API here?  Isn't this just `qt_compat.QtWidgets.QApplication.instance()`?  (which is standard qt, doesn't need to be documented, yada yada)
(I guess I would rather deprecate backend_qt.qApp instead.)

Reply:
Because we used to expose it as public and I know for a fact (because it bit us at NSLS-II!) that it is being used in the wild.

Reply:
Do you want to block https://github.com/matplotlib/matplotlib/pull/22503 then?  (In particular, note the subtly different semantics.)  (It's OK if you want to block it, just let me know and I'll close that :))

Reply:
I am 👍🏻 on deprecating it, but need to put it back to do so 😞